application/common/models/Password.php

```diff
@@ -214,5 +222,64 @@ public function save()
         }
     }
 
+    public function validateNotPublicPassword($attribute)
+    {
+        /*
+         * block passwords provided in https://www.youtube.com/watch?v=3LRnjzA8auk&list=PLu5OsENIeX656zXJ96FCL169WNmvnPveo
```

Comment:
Minor issue, but it's probably worth wrapping just before the URL to avoid going over 120 chars on that line.

Reply:
could also look for a shortened url...and I think this video id has changed, just a moment...

Reply:
se this `https://youtu.be/WTMZYuoztoM`